core/templates/services/translations-backend-api.service.ts

```diff
@@ -37,6 +37,9 @@ export class TranslationsBackendApiService {
     return this.http.get<TranslationsDict>(
       `${this.prefix}${languageCode}${this.suffix}`).toPromise();
   }
+  loadTranslationFileHash(url: string): Promise<String> {
```

Comment:
```suggestion
  async loadTranslationFileHashAsync(url: string): Promise<String> {
```

(Though it's not expected from this PR but we are expecting to have this pattern for async functions.)

Reply:
Done!